src/views/AuthenticationWrapper/DevSecurity.tsx

```diff
@@ -0,0 +1,152 @@
+import React, { ReactEventHandler, useEffect, useRef, useState } from 'react';
+import { OktaContext } from '@okta/okta-react';
+
+const storageKey = 'dev-user-config';
+
+const initialAuthState = {
+  isAuthenticated: false,
+  isPending: false,
+  name: '',
+  euaId: '',
+  groups: [] as string[]
+};
+
+const jobCodes = ['EASI_D_GOVTEAM', 'EASI_P_GOVTEAM'];
+
+type ParentComponentProps = {
+  children: React.ReactNode;
+};
+
+const DevSecurity = ({ children }: ParentComponentProps) => {
+  const [authState, setAuthState] = useState(initialAuthState);
+  const [euaId, setEuaId] = useState('');
+  const checkboxValues = useRef(new Set<string>());
+
+  const authService = {
+    login: () => {},
+    logout: () => {
+      window.localStorage.removeItem(storageKey);
+      window.location.href = '/';
+    },
+    getUser: () => {
+      return Promise.resolve({ name: authState.name });
+    },
+    getTokenManager: () => {
+      return {
+        off: () => {},
+        on: () => {},
+        renew: () => {}
+      };
+    }
+  };
+
+  useEffect(() => {
+    if (window.localStorage[storageKey]) {
+      const state = JSON.parse(window.localStorage[storageKey]);
+      setAuthState(as => {
+        return {
+          ...as,
+          name: `User ${state.eua}`,
+          isAuthenticated: true
+        };
+      });
+    }
+  }, []);
+
+  const handleSubmit: ReactEventHandler = event => {
+    event.preventDefault();
+    const value = {
+      eua: euaId,
+      jobCodes: Array.from(checkboxValues.current)
+    };
+    localStorage.setItem(storageKey, JSON.stringify(value));
+    localStorage.removeItem('okta-token-storage'); // ensure that the dev token is used
+    setAuthState({
+      ...authState,
+      isAuthenticated: true,
+      name: `User ${euaId}`,
+      euaId,
+      groups: Array.from(checkboxValues.current)
+    });
+  };
+
+  const checkboxChange: ReactEventHandler<HTMLInputElement> = event => {
+    if (event.currentTarget.checked) {
+      checkboxValues.current.add(event.currentTarget.value);
+    } else {
+      checkboxValues.current.delete(event.currentTarget.value);
+    }
+  };
+
+  return authState.isAuthenticated ? (
+    <OktaContext.Provider value={{ authService, authState }}>
+      {children}
+    </OktaContext.Provider>
+  ) : (
```

Comment:
Should the login render be in this `DevSecurity` component? Should we have it show up on `/login` on `local` environments? Or a login route that only shows for `local` environments. It feels like this wrapper has different responsibilities on different environments.